src/app/work-item/work-item-detail/work-item-detail.component.ts

```diff
@@ -324,7 +324,7 @@ export class WorkItemDetailComponent implements OnInit, AfterViewInit, OnDestroy
   getAllUsers(): Observable<any> {
     return Observable.combineLatest(
       this.userService.getUser(),
-      this.userService.getAllUsers()
+      this.workItemService.getCollaborators()
```

Comment:
@sanbornsen I understand, feel free to clone my branch( this PR ) and make changes here directly. ( No need to open a  PR on this PR/branch )